examples/classifier.c

```diff
@@ -415,7 +442,7 @@ float** get_predictions (network* net, char** paths, char** labels, int m, int c
 
         image im = load_image_color(paths[i], 0, 0);
         image crop = center_crop_image(im, net->w, net->h);
-        
+
         float *pred = network_predict(net, crop.data);
 
         if(net->hierarchy) hierarchy_predictions(pred, net->outputs, net->hierarchy, 1, 1);
```

Comment:
Is there anywhere else you should use `shuffled_indices[i]`?

Reply:
corrigido